Dispatchfile

```diff
@@ -201,6 +201,7 @@ action(tasks=[do_bump_addon], on=push(branches=["master"], paths=["!" + kommande
 action(tasks=["test-helm2", "test-helm3"], on=pull_request(chatops=["test"]))
 action(tasks=["test-helm2"], on=pull_request(chatops=["test-helm2"]))
 action(tasks=["test-helm3"], on=pull_request(chatops=["test-helm3"]))
+action(tasks=[do_bump_kommander_catalog_api], on=pull_request(chatops=["bump-kommander"]))
```

Comment:
should the chatop be more accurate? you're not bumping kommander but kommander catalog api